src/styles/LandingPageStyles/index.less

```diff
@@ -320,6 +316,28 @@
   }
 }
 
+//** Landing page styles - Tablet fix **//
```

Comment:
Really meticulous styling, target by target and line by line pretty much. The format now shows a lot of work was put into it! Nice work on responsiveness!